script/run-isolated-tests.sh

```diff
@@ -5,10 +5,17 @@
 APP_SUBPATHS="$(find src -name manifest.json | sed -E 's/\/manifest.json//g' | xargs | sed 's/ /|/g')"
 
 # Match changed files to the app paths determined above
-FILES_CHANGED="$(git diff --name-only master)"
-APP_SUBPATHS_CHANGED="$(echo $FILES_CHANGED | grep -oE "($APP_SUBPATHS)")"
+FILES_CHANGED="$(git diff --name-only HEAD $(git merge-base HEAD origin/master))"
```

Comment:
So this diffs the tracked changes on the current branch against the commit where the current branch branched off of master.  I think that's good.

This syntax does the same thing but may be a bit clearer:

```
git diff master...HEAD --name-only
```

Reply:
Oh, neat!